web/web/api/v1/login.py

```diff
@@ -0,0 +1,164 @@
+from flask import request, jsonify, Blueprint
+from datetime import timedelta
+from werkzeug.security import generate_password_hash
+from python_http_client.exceptions import BadRequestsError
+from marshmallow import ValidationError
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm.exc import NoResultFound, MultipleResultsFound
+from flask_jwt_extended import (create_access_token, get_jwt_identity,
+                                create_refresh_token, get_jti)
+from .response_wrapper import ApiResponseWrapper
+from web.database import db
+from web.models.login import Login, LoginSchema
+from web.models.user import User, UserSchema
+from web.redis import revoked_store
+
+
+login_api_bp = Blueprint('login_api_bp', __name__)
+
+
+@login_api_bp.route('/login/<int:login_id>', methods=['GET'])
+def show_login_info(login_id):
+    '''
+    Retrieves one login object
+    '''
+    arw = ApiResponseWrapper()
+    login_schema = LoginSchema(exclude=['password_hash'])
+
+    try:
+        login = Login.query.filter_by(login_id=login_id).one()
+
+    except (MultipleResultsFound, NoResultFound):
+        arw.add_errors('No result found or multiple results found')
+
+    if arw.has_errors():
+        return arw.to_json(None, 400)
+
+    results = login_schema.dump(login)
+
+    return arw.to_json(results)
+
+
+@login_api_bp.route('/login/<int:login_id>', methods=['PUT'])
+def modify_login(login_id):
+    '''
+    Updates one login object in database
```

Comment:
It seems the only data that would ever change is the password hash (and the `updated_at` due to a change in password). I think it would be better to be explicit and call this method update password and expect a new password (and maybe the old one to verify.)
---
May also be worth creating a delete account endpoint which essentially sets the password_hash to null, which is maybe how we'd represent a disabled account....? Thoughts?

Reply:
I think that a github bug, I have no idea why that comment is large and bolded

Reply:
Yes, will fix for update password endpoint instead of general login put endpoint. Yes I think a delete endpoint would be useful. Alternative would be to add is_active column on the login table like in the user table.. this column is currently not on the login table?

Reply:
Yeah, this is an interesting one, the User model has `is_active` and it should be coordinated with the Login model. Meaning, a user that is inactive, should not be able to login regardless of the Login model's flag. I think we can do w/o a `is_active` on the Login. What do you think?

Reply:
So I just created an reset password endpoint with PATCH, I didn't add verifying the old password because I feel like most cases are because they may have forgotten their old password. I will do a different PR to follow through with a workflow for forgotten password, etc. I think you're right we can circle back to user.is_active through the relationship so it's not needed on login. Is the delete endpoint necessary ?

Reply:
I don't think delete is necessary on login.